ingester/src/querier_handler.rs

```diff
@@ -72,31 +85,92 @@ pub async fn prepare_data_to_querier(
     ingest_data: &Arc<IngesterData>,
     request: &IngesterQueryRequest,
 ) -> Result<IngesterQueryResponse> {
+    let mut schema_merger = SchemaMerger::new();
+    let mut unpersisted_partitions = BTreeMap::new();
+    let mut batches = vec![];
+    for sequencer_data in ingest_data.sequencers.values() {
+        let namespace_data = match sequencer_data.namespace(&request.namespace) {
+            Some(namespace_data) => namespace_data,
+            None => {
+                continue;
+            }
+        };
+
+        let table_data = match namespace_data.table_data(&request.table) {
+            Some(table_data) => table_data,
+            None => {
+                continue;
+            }
+        };
```

Comment:
I wonder if writing this code like would be nicer

```suggestion
        let maybe_table_data = sequencer_data.namespace(&request.namespace)
          .map(|namespace_data| namespace_data.table_data(&request.table);
     
          match maybe_table_data {
            Some(table_data) => table_data,
            None => {
                continue;
            }
        };     
```